app/Platform/Jobs/UpdatePlayerMetricsJob.php

```diff
@@ -25,6 +27,17 @@ public function __construct(
 
     public function handle(): void
     {
+        if ($this->batch()?->cancelled()) {
+            return;
+        }
+
+        $user = User::find($this->userId);
+
+        if (!$user) {
+            // might've been deleted
```

Comment:
nit: It may be worth logging this.

Reply:
ideally we should not ever run into this - this check is to be removed, too.
the only reason i added it is because the player_games table is not yet properly cleaned up after account deletions. i have a draft PR up for that.